doc/lsp.txt

```diff
@@ -507,6 +507,10 @@ outlineOnRight		|Boolean| option.  Open the outline window on the
 						*lsp-opt-outlineWinSize*
 outlineWinSize		|Number| option.  The size of the symbol Outline
 			window.  By default this is set to 20.
+						*lsp-opt-propOverride*
+propOverride		|Boolean| option.  when TRUE textprops highlight
+			overrides any other, including 'cursorline' and Visual
+			By default this is set to false.
```

Comment:
if make it judge/work with `cursorlineopt` then no this option was required, it is 'ture' when no cursorline.
but simply it is ok, i think we can live with this option.